### PR TITLE
[ui][fix] fix support page tests to reflect UI update

### PR DESCRIPTION
**Pull request by mcada**
//test: `@support-page`

### CODE REVIEW

File: ui-tests/src/test/java/io/syndesis/qe/steps/CommonSteps.java [hunk 1]
```diff
@@ -324,17 +323,23 @@ public void navigateTo(String title) {
...
+        SelenideElement dropdownElementsTable = $(By.className("pf-c-dropdown__menu")).shouldBe(visible);
         ElementsCollection dropdownElements = dropdownElementsTable.findAll(By.tagName("a"))
                 .shouldBe(CollectionCondition.sizeGreaterThanOrEqual(1));
 
         dropdownElements.filter(text(title)).shouldHaveSize(1).get(0).shouldBe(visible).click();
+
+        //close the help menu
+        if (helpDropdownMenu.parent().$$(By.className("pf-c-dropdown__menu")).size() >= 1) {
+            helpDropdownMenu.click();
+        }
```
Comment (avano) [inline ui-tests/src/test/java/io/syndesis/qe/steps/CommonSteps.java]: maybe add a todo here to remove it when the issue is fixed?

Comment (mcada) [reply ui-tests/src/test/java/io/syndesis/qe/steps/CommonSteps.java]: Well, it wont do anything after it gets fixed so it doesn't matter... But sure, why not.

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mcada** [comment]
failed test is from rest smokes, rdy to merge

**Comment by avano** [comment]
@mcada not really: 

```
Test Result (1 failure / +1)
Support page.Export diagnostic of single integration
```

**Comment by mcada** [comment]
:O I swear when I clicked into jenkins results I saw UI ok and rest failed, must have missed tab or something, thx for the recheck

**Comment by mcada** [comment]
It works locally anyway, there is no reason for data mapper to not be visible here.

**Comment by mcada** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mcada** [comment]
syndesis failed to build...

**Comment by mcada** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Pull request merged**